common/src/main/res/values/strings.xml

```diff
@@ -1299,4 +1299,8 @@
     <string name="sensor_description_car_speed">The current speed of the car</string>
     <string name="basic_sensor_name_car_range_remaining">Car range remaining</string>
     <string name="sensor_description_car_range_remaining">The range remaining for the car in meters</string>
+    <string name="feature_flag">Feature flag</string>
+    <string name="feature_flag_summary">Manage feature flags for the app (advanced users only). Proceed with caution, as this area contains experimental features that may affect stability.</string>
+    <string name="string_feature_flag_update">Update</string>
+    <string name="string_feature_flag_cancel">Cancel</string>
```

Comment:
I don't think I can add this new keys myself in Lokalise.

Reply:
If your PR is accepted and merged, Lokalise will pick up the new strings automatically